docs/source/api/v2/deliveryservices_sslkeys_generate_letsencrypt.rst

```diff
@@ -30,6 +30,10 @@ Generates an SSL certificate and private key using Let's Encrypt for a :term:`De
 Request Structure
 -----------------
 :key:             The :ref:`ds-xmlid` of the :term:`Delivery Service` for which keys will be generated
+:deliveryservice: The :ref:`ds-xmlid` of the :term:`Delivery Service` for which keys will be generated
+
+.. NOTE:: Either the ``key`` or the ``deliveryservice`` field must be provided. If both are provided, then they must match.
```

Comment:
This should probably be a footnote instead, because it applies to multiple things and the way it is now it's breaking the list. So the generated output is something like:
```
<field list>
  <list item>...</list item>
  <list item>...</list item>
</field list>
<admonition>...</admonition>
<field list>
  <list item>...</list item>
  <list item>...</list item>
</field list>
```
so things aren't all part of the same list. But they oughtta be.

Reply:
done